qc-runner/src/cellSizeDistribution.r

```diff
@@ -103,7 +103,21 @@ task <- function(seurat_obj, config, task_name, sample_id, num_cells_to_downsamp
     if (exists('auto', where=config)){
         if (as.logical(toupper(config$auto)))
             # config not really needed for this one (maybe later for threshold.low/high):
-            minCellSize <- generate_default_values_cellSizeDistribution(sample_subset, config)
+            if (exists('auto', where=config)){
```

Comment:
We are checking the existence of auto but we just accessed it in the previous line

Reply:
Yes, by mistake I duplicated that part of code